llvm/test/Transforms/LoopVectorize/AArch64/sve2-histcnt.ll

```diff
@@ -0,0 +1,82 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 3
+; RUN: opt < %s -passes=loop-vectorize -force-vector-interleave=1 -force-target-instruction-cost=1 -S | FileCheck %s
```

Comment:
Eventually we want this to be controlled using only `-aarch64-base-histcnt-cost=1` ?

Reply:
We can add a new flag for histcnt cost, but it's actually the gathers and scatters which prevent vectorization here. I've changed to use -sve-gather-overhead and -sve-scatter-overhead instead.

I do think we've made gathers and scatters too expensive by default for SVE, but we'll need to spend some time figuring out which loops get worse with it on and working out a better model.

Reply:
Yep, sounds reasonable. Thanks.